pluginClientSdkLib/build.gradle

```diff
@@ -57,6 +58,14 @@ android.libraryVariants.all { variant ->
     }
 }
 
+if (JavaVersion.current().isJava8Compatible()) {
```

Comment:
I think both of those can be removed, I'll update later.

On Sun, Mar 15, 2015, 4:16 PM Carter Jernigan <notifications@github.com>
wrote:

> In pluginClientSdkLib/build.gradle
> <https://github.com/twofortyfouram/android-plugin-client-sdk-for-locale/pull/1#discussion_r26451749>
> :
>
> > @@ -57,6 +58,14 @@ android.libraryVariants.all { variant ->
> >      }
> >  }
> >
> > +if (JavaVersion.current().isJava8Compatible()) {
>
> What problem were you running into that needed this option to be set? When
> building locally, I don't see any warnings with this block removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twofortyfouram/android-plugin-client-sdk-for-locale/pull/1/files#r26451749>
> .
>